sql/core/src/test/scala/org/apache/spark/sql/internal/SharedStateSuite.scala

```diff
@@ -52,15 +52,4 @@ class SharedStateSuite extends SharedSparkSession {
     assert(conf.isInstanceOf[Configuration])
     assert(conf.asInstanceOf[Configuration].get("fs.defaultFS") == "file:///")
   }
-
-  test("SPARK-33740: hadoop configs in hive-site.xml can overrides pre-existing hadoop ones") {
```

Comment:
Is this removed because this is merged to the new test coverage?

Reply:
yes, the newly added case will take over this one